libraries/browser-tracker-core/src/tracker/types.ts

```diff
@@ -661,3 +631,33 @@ export interface ClientSession extends Record<string, unknown> {
    */
   firstEventTimestamp: string | null;
 }
+
+/**
+ * A collection of GET events which are sent to the collector.
+ * This will be a collection of query strings.
+ */
+export type GetBatch = string[];
+
+/**
+ * A collection of POST events which are sent to the collector.
+ * This will be a collection of JSON objects.
+ */
+export type PostBatch = Record<string, unknown>[];
+
+/**
+ * A collection of events which are sent to the collector.
+ * This can either be a collection of query strings or JSON objects.
+ */
+export type EventBatch = GetBatch | PostBatch;
```

Comment:
To be honest, I am a bit skeptical about the usefulness of this information in the success and failure callbacks – what can users do with it? But maybe I'm just being a grumpy old man and probably it's better than nothing :)

Reply:
After discussion on Slack - the event batches will be staying.